### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Kenny2github — CHANGES_REQUESTED** [review]
The rest needs to be translated: https://github.com/Kenny2github/kenny2automate/tree/master/i18n/8ball https://github.com/Kenny2github/kenny2automate/tree/master/i18n/battleship https://github.com/Kenny2github/kenny2automate/tree/master/i18n/card_games https://github.com/Kenny2github/kenny2automate/tree/master/i18n/connect4 https://github.com/Kenny2github/kenny2automate/tree/master/i18n/games https://github.com/Kenny2github/kenny2automate/tree/master/i18n/hangman https://github.com/Kenny2github/kenny2automate/tree/master/i18n/pm_games https://github.com/Kenny2github/kenny2automate/tree/master/i18n/regexes https://github.com/Kenny2github/kenny2automate/tree/master/i18n/scratch https://github.com/Kenny2github/kenny2automate/tree/master/i18n/wiki

**Review by Kenny2github — APPROVED** [review]
I can't review the translations, but this looks good to go! However, I have some changes coming up that will need translation updates, so I won't merge this yet until those changes are finished - I'll ask you to add the new translations once they're done.

**Review by Kenny2github — CHANGES_REQUESTED** [review]
The commit has come in now. Please update all of these translations: [battleship](https://github.com/Kenny2github/kenny2automate/commit/34021b8d071b3114e8ff5ed4881c650d97c3cbb3#diff-3eb18ef4ec6e1edb2bb0553ccfebbf90) [card_games](https://github.com/Kenny2github/kenny2automate/commit/34021b8d071b3114e8ff5ed4881c650d97c3cbb3#diff-088b47ef74d67c868b33ed05645f138d) [pm_games](https://github.com/Kenny2github/kenny2automate/commit/34021b8d071b3114e8ff5ed4881c650d97c3cbb3#diff-330c0aa6934b7b9d90e647d3de81219c)

**Comment by ghost** [comment]
Done

File: i18n/battleship/ru.json [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+  "edit-instructions": "Отправьте сообщение, содержащее точный контент:",
   "fire-instructions": "Отправьте координаты (например A0 или 3b) чтобы выстрелить туда.",
   "wait-instructions": "Ожадание, пока другие игроки выстелят…",
   "won-title": "Вы победили!",
   "won": "Поздравляю! Вы потопили все корабли соперника!",
   "lost-title": "Вы проиграли!",
   "lost": "Ваш соперник потопил все ваши корабли!"
-}
\ No newline at end of file
+  "hitboard": "Доска",
```
Comment (Kenny2github) [inline i18n/battleship/ru.json]: This is invalid JSON, the line above needs a comma (`,`) at the end.

File: i18n/battleship/ru.json [hunk 2]
```diff
@@ -1,9 +1,18 @@
 {
   "place-instructions": "Нажмите кнопку **дважды**, чтобы переместить корабль; дважды щелкните, когда закончите.",
+  "edit-instructions": "Отправьте сообщение, содержащее точный контент:",
   "fire-instructions": "Отправьте координаты (например A0 или 3b) чтобы выстрелить туда.",
```
Comment (Kenny2github) [inline i18n/battleship/ru.json]: This line was [changed](https://github.com/Kenny2github/kenny2automate/commit/34021b8d071b3114e8ff5ed4881c650d97c3cbb3#diff-3eb18ef4ec6e1edb2bb0553ccfebbf90R4) in the commit, please update it as well.

**Review by Kenny2github — CHANGES_REQUESTED** [review]
Some changes needed.

(for some reason GitHub thinks I'm reviewing outdated commits; I'm not, but whatever)

**Comment by ghost** [comment]
Done.

**Review by Kenny2github — APPROVED** [review]

**Pull request merged**